route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManager.cs

```diff
@@ -4847,6 +4849,28 @@ public bool RemoveOrders(string[] orderIds, out string errorString)
             return GetJsonObjectFromAPIAsync<DataObject>(request, R4MEInfrastructureSettings.ApiHost, HttpMethodType.Put, null, false, false);
         }
 
+        /// <summary>
+        ///     Transfer an order to other organization
+        /// </summary>
+        /// <param name="transferredOrder"> An order to transfer </param>
+        /// <param name="organizationApiKey">An API key of a destination organization</param>
+        /// <returns>Transferred order</returns>
+        public Order TransferOrderToOrganization(Order transferredOrder, string organizationApiKey, out string errorString)
```

Comment:
please, add async version of that method too

Reply:
Ok, I'll add, but this method assumes to send  only one order and in this case rarely is used asynchronous methods.